tests/conftest.py

```diff
@@ -28,22 +28,8 @@ def threadsafe_data_dir(tmp_path_factory) -> Path:
     yield data_dir
 
 
```

Comment:
```suggestion

@pytest.fixture(autouse=True, scope="session")
def nimbus(threadsafe_data_dir, worker_id) -> pooch.Pooch:
    kwargs = {}
    if worker_id != "master":
        kwargs["cache_dir"] = threadsafe_data_dir
    return _nimbus(**kwargs)
```

Reply:
This function builds a class with all the default settings from `xclim`. There shouldn't be any reason for a developer of `xhydro` to change these. 

It should go without saying, but this would then be used further down and within the testing suite.